src/app/containers/Gist/index.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import Gist from '.';
+import fixtureData from './fixtures';
+
+// eslint-disable-next-line react/prop-types
+const GistWithContext = ({ gistData = fixtureData }) => (
+  <ServiceContextProvider dir="ltr" service="news" lang="en-GB">
+    <Gist gistData={gistData} />
+  </ServiceContextProvider>
+);
+
+describe('Gist', () => {
+  it('should render the gist heading', () => {
+    const { getByText } = render(<GistWithContext />);
```

Comment:
Should we add a couple snapshot tests here? Maybe one with a single listItem and another with multiple?